inside_docker/maf_scripts.sh

```diff
@@ -58,8 +62,6 @@ maf_lengths() {
     done
 }
 
-
-
```

Comment:
You have a ton of trailing whitespace. Most editors can be configured to automatically trim this. If you prefer to do it manually that's fine too.

In vim, you can see trailing whitespace a few ways. One option is to searching (`/`) for "space, newline", e.g. ` \n` and having `set hlsearch` to highlight the selection. You can remove them all with "next, change in word, ctrl-C", or `nciw^c`, and repeating over and over with "next, repeat", or `n.n.n.n.n.n.`...

The other way to see it is to have the editor highlight it for you. In vim, my preferred way for this is with the following line of code in your `.vimrc` https://github.com/mweitzel/dotfiles/blob/master/.vimrc#L46. Be careful copy/pasting it, as the "tab" character doesn't always work easily with clipboards.